webui/react/src/components/Table.tsx

```diff
@@ -132,7 +132,11 @@ export const taskNameRenderer: TaskRenderer = (id, record) => (
   <div>
     <ConditionalWrapper
       condition={canBeOpened(record)}
-      wrapper={(ch) => <Link path={paths.interactive(record)}>{ch}</Link>}>
+      wrapper={(ch) => (
+        <a href={paths.interactive(record)} rel="noopener noreferrer" target="_blank">
```

Comment:
Used `href` instead of using `openBlank` func cuz we cannot see the lint when cursor is hovered on the link on the bottom left.